src/renderer/services/wallet/util.test.ts

```diff
@@ -173,4 +174,32 @@ describe('services/wallet/util/', () => {
       ])
     })
   })
+
+  describe('getBalanceByAsset', () => {
+    it('get balance by asset', () => {
+      expect(
+        getBalanceByAsset(AssetBNB)([
+          {
+            asset: ASSETS_TESTNET.TOMO
+          },
+          {
+            asset: ASSETS_TESTNET.BOLT
+          },
+          {
+            asset: ASSETS_TESTNET.FTM
+          },
+          {
+            asset: ASSETS_TESTNET.BNB
+          },
+          {
+            asset: ASSETS_TESTNET.RUNE
```

Comment:
Pls don't use `ASSETS_TESTNET.RUNE` anymore - it will be deprecated in the very near future. Use `AssetRuneNative`.